languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -145738,6 +145738,22 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                 <url>https://fr.wikipedia.org/wiki/Champigny-sur-Marne</url>
                 <example correction="Champigny-sur-Marne">Il habite à <marker>Champigny sur Marne</marker>.</example>
             </rule>
+            <rule>
+                <antipattern>
+                    <token skip="-1">plus</token>
+                    <token>Hauts</token>
+                    <token>de</token>
+                    <token>France</token>
+                </antipattern>
+                <pattern>
+                    <token>Hauts</token>
+                    <token>de</token>
+                    <token>France</token>
+                </pattern>
+                <message>Cette région s'écrit avec un trait d'union</message>
+                <suggestion>Hauts-de-France</suggestion>
+                <example correction="Hauts-de-France">Les  <marker>Hauts de France</marker> sont une région française.</example>
```

Comment:
chore: I think there's a double space in there `"Hauts-de-France">Les  <marker>Hauts de France` 🥜
question: is this pattern doing something for the exact structure in the original issue, _Les hauts de France sont une région française._ with hauts in lowercase?

Reply:
I think the branch thing is gonna be okay (I hope. Ahah).
Yeah, we should temp_off just as usual, even if I agree, this looks safe. Just in case (sometimes the nighlty brings out weird stuff).

Reply:
Alright, the rule is in temp_off now, and I think ready to merge